app/models/incident.rb

```diff
@@ -104,43 +104,53 @@ def csv_row
     row
   end
 
-  def export_claims_xml!
+  def export_claims_xml(current_user)
     report = driver_incident_report
     builder = Nokogiri::XML::Builder.new do |doc|
       doc.claims_incident_data do
-        doc.incident do
```

Comment:
So we're removing <incident> and replacing it with just <drivers_report> ?

Reply:
Yes, I asked Liza which values went into incident and driver report respectively, and she said that all values are within driver report in claims.